scripts/ci/deploy_condition.sh

```diff
@@ -3,8 +3,16 @@
 set -e
 
 if [[ $TRAVIS_OS_NAME == "osx" && $TRAVIS_OSX_IMAGE != "xcode8.3" ]]; then
-    echo false
-    exit 0
+    exit 1
+fi
+
+if [[ $TRAVIS_EVENT_TYPE = pull_request || $TRAVIS_EVENT_TPYE = cron ]]; then
+    exit 2
+fi
+
+# ensure at least one positional arg exists
```

Comment:
what's the reason behind this one, @casperdcl ?

Reply:
travis' `deploy.file` might not exist. This is a convenient way to check

Reply:
@casperdcl Still don't understand what it is for. What is `deploy.file`?

Reply:
Got it. It is because of globs passed to it.